src/Sylius/Behat/spec/Context/Ui/Admin/ManagingCountriesContextSpec.php

```diff
@@ -212,4 +215,22 @@ function it_throws_not_equal_exception_if_country_code_field_is_not_disabled(Upd
         $countryUpdatePage->isCodeFieldDisabled()->willReturn(false);
         $this->shouldThrow(\InvalidArgumentException::class)->during('theCodeFieldShouldBeDisabled');
     }
+
+    function it_asserts_that_a_country_has_the_province(CountryInterface $country, UpdatePageInterface $countryUpdatePage)
+    {
+        $country->getId()->willReturn(1);
+        $countryUpdatePage->isOpen(['id' => 1])->willReturn(true);
+        $countryUpdatePage->isThereProvince('Scotland')->willReturn(true);
+
+        $this->countryShouldHaveProvince($country, 'Scotland');
```

Comment:
According to standards (https://en.wikipedia.org/wiki/ISO_3166-2:GB) Scotland is a subdivision of the United Kingdom.